server/repository/src/db_diesel/changelog/changelog.rs

```diff
@@ -201,7 +201,7 @@ impl ChangelogTableName {
             ChangelogTableName::Report => ChangeLogSyncStyle::Central,
             ChangelogTableName::FormSchema => ChangeLogSyncStyle::Central,
             ChangelogTableName::PluginData => ChangeLogSyncStyle::RemoteAndCentral,
-            ChangelogTableName::Preference => ChangeLogSyncStyle::Central,
+            ChangelogTableName::Preference => ChangeLogSyncStyle::RemoteAndCentral,
```

Comment:
before was syncing all prefs to all stores - you should only receive global prefs, or store prefs for stores on your site